File: code-examples/java/carts/patch.java
HttpResponse<String> response = Unirest.patch("http://api.marketcloud.it/v0/carts/:id")
  .header("authorization", "YourPublicKey:YourToken")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .body("{\"op\" : \"update\",\"items\" : [{\"product_id\":1,\"quantity\":1}]}")
  .asString();

File: code-examples/go/carts/patch.go
package main

import (
	"fmt"
	"strings"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://api.marketcloud.it/v0/carts/9"

	payload := strings.NewReader("{\"op\" : \"update\",\"items\" : [{\"product_id\":13,\"quantity\":20},{\"product_id\":15,\"quantity\":18}]}")

	req, _ := http.NewRequest("PATCH", url, payload)

	req.Header.Add("authorization", "YourPublicKey:YourToken")
	req.Header.Add("accept", "application/json")
	req.Header.Add("content-type", "application/json")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}